Userland/Applications/PixelPaint/PixelPaintWindow.gml

```diff
@@ -43,59 +43,83 @@
             }
         }
 
-        @GUI::Widget {
-            fill_with_background_color: true
+        @GUI::DynamicWidgetContainer {
+            section_label: "Tools"
+            config_domain: "PixelPaint"
+            with_individual_order: true
             fixed_width: 200
-            layout: @GUI::VerticalBoxLayout {}
-
-            @GUI::GroupBox {
-                title: "Layers"
-                layout: @GUI::VerticalBoxLayout {
-                    margins: [6]
+            preferred_height: "grow"
+            detached_size: [200, 640]
```

Comment:
I don't think this is necessary if we carry over the in-window widget size correctly, as pointed out in an earlier comment.

Reply:
I've changed the logic so that the new window defaults to the in-window size, but having an alternative other detached size seems quite comfortable as you might want to detach the widget because it should display more information when more space is available - so you don't need to resize the window manually after detaching...